Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+[^DE_EM2DX]: [German Emergency 2 Deluxe](waiting on null to upload/)
+[^Petterson]: [Petterson und Findus - Hühnertanz und Katerzirkus](waiting on null to upload)
+[^Kenny]: [Kenny's Adventure](waiting on null to upload)
+[^101786]: [Max and the Magician (Europe) (En,Fr,De,Es)](http://redump.org/disc/101786/)
+
+#### copy-X light:
+
+Basic ring protection via dummy files intersected by rings. No software protection is used unless otherwise supplied by the game itself. Can be circumvented by simply not copying the dummy files when copying the rest of the game from the disc.
+
+#### copy-X professional:
+
+Builds upon copy-X light. Contains everything described there, but also adds software protection in the form of a disc check, which appears to additionally check one of the dummy files. Copy-X professional also checks if the contemporaneously-popular kernel debugging software SoftICE is active via a process called SoftICE-Test, checking both for "SoftICE for Windows 95" and for "SoftICE for Windows NT", ending game execution if detected when booting the game.
+
+#### Untested:
+
+* copy-X audio[^D_8232013][^D_1436646][^D_2182905][^D_451419]
+* copy-X trial maker
+
+[^D_8232013]: [Die Wilden Kerle – DWK4 - Der Angriff Der Silberlichten (Das Hörspiel Zum Kinofilm)](https://www.discogs.com/release/8232013-Die-Wilden-Kerle-DWK4-Der-Angriff-Der-Silberlichten-Das-H%C3%B6rspiel-Zum-Kinofilm)
+[^D_1436646]: [Laava – Wherever You Are (I Feel Love)](https://www.discogs.com/release/1436646-Laava-Wherever-You-Are-I-Feel-Love)
+[^D_2182905]: [Manfred Mann* – 2006](https://www.discogs.com/release/2182905-Manfred-Mann-2006)
+[^D_451419]: [De-Phazz – Godsdog](https://www.discogs.com/release/451419-De-Phazz-Godsdog)
+
+#### copy-X audio:
+
+No samples have been tested at the moment, so exact specifics cannot be confirmed. Used to protect Audio CDs.
+
+#### copy-X trial maker:
+
+No samples currently known, and thus no information either.
+
+
+***
+
+## Samples:
+ 
+#### Confirmed:
+
+* copy-X light[^84759][^107929][^81628][^Kenny][^101786]
+* copy-X professional[^108150][^48393][^82475][^DE_EM2DX][^Petterson]
+
+#### Unconfirmed: 
+
+* copy-X audio[^D_8232013][^D_1436646][^D_2182905][^D_451419]
+* German Westerner 2[^pcgames_de_westerner2][^gamestar_de_westerner2]
+* Many games from Sixteen Tons Entertainment
+* Many games from Tivola
+* Many games from Terzio
+* Many pieces of software from Magix 
+
+[^pcgames_de_westerner2]: [The Westerner 2 im PC-Games-Test: "Ein enttäuschendes Adventure ... " ](https://www.pcgames.de/The-Westerner-2-Fenimore-Fillmores-Revenge-Spiel-38466/Tests/The-Westerner-2-im-PC-Games-Test-Ein-enttaeuschendes-Adventure-679282/)
+[^gamestar_de_westerner2]: [The Westerner 2: Westernheld Fenimore Fillmore ist er-
+wachsen geworden. Im Gegenzug
+schrumpft die Spielzeit auf Babyformat.](https://download.gamestar.de/public/43400/43452/05_2009_096.pdf)
+
+***
+
+## Supported Platforms:
```

Comment:
Please refer to these as their full names (such as "copy-X light") to make the lack of capitalization on version names stand out less.

Reply:
Fixed.